resources/datasets/hmda/generate_msamd_csv.sh

```diff
@@ -151,12 +159,24 @@ sed_tweaks='
 
 if [ $INPUT_TYPE = 'xlsx' ]; then
     # xlsx2csv gracefully handles unicode nastiness, thankfully.
-    dump_cmd="xlsx2csv -d '|' \"$1\""
+    dump_cmd="xlsx2csv -d '|' \"$INPUT_FILE\""
+elif [ $INPUT_TYPE = 'tab' ]; then
+    # this was specifically made for FFIEC files https://www.ffiec.gov/hmda/hmdaflat.htm 
+    # (https://www.ffiec.gov/hmdarawdata/OTHER/2007HMDAMSADescription.zip)
+    # Convert DOS end of line characters with characters awk will properly handle
+    dos2unix $INPUT_FILE
```

Comment:
I didn't catch this earlier...in general I think it's bad form to modify the input file.  We can use the following style to apply the changes inline:

```shell
dump_cmd="dos2unix < \"$INPUT_FILE\" | $sed_bin 's/\t/|/'"
```

Of course this makes it harder to see if the program is installed...but you can do that similar to the other places in this program:

```shell
command -v dos2unix 2>/dev/null || {
}
```

I worry that we're overcomplicating things though.  A much easier solution would be to replace dos2unix with this one-liner in sed_tweaks:
```shell
# dos2unix
s/\r//
```